requirements-integration.txt

```diff
@@ -0,0 +1,3 @@
+ixbrl-viewer==1.4.10
```

Comment:
Do you have plans to align this with #990 so we don't have this dependency in two different files? 
Maybe name this `requirements-plugins.txt` and drop `-r requirements-dev.txt` so it can be required by `requirements-build.txt`?